src/run.py

```diff
@@ -60,37 +59,46 @@ def get_sess_dirs(subj_dir_path, mapping_dict):
     return sess_dirs
 
 
+def construct_vqcdeface_cmd(qc_dir):
+    rel_paths_to_orig = [re.sub('/orig.nii.gz', '', str(o.relative_to(qc_dir))) for o in qc_dir.rglob('orig.nii.gz')]
+    with open(qc_dir / 'defacing_id_list.txt', 'w') as f:
+        f.write('\n'.join(rel_paths_to_orig))
+
+    vqcdeface_cmd = f"vqcdeface -u {qc_dir} -i {qc_dir / 'defacing_id_list.txt'} -m orig.nii.gz -d defaced.nii.gz -r defaced_render"
+
+    return vqcdeface_cmd
+
+
 def main():
     # get command line arguments
     args = get_args()
 
-    input_dir = Path(args.bids_dir).resolve()
-    output = Path(args.output_dir).resolve()
+    input_dir = args.bids_dir.resolve()
+    output = args.output_dir.resolve()
     no_clean = args.no_clean
 
     to_deface = []
     # for one subject or list of subjects (and all their sessions, if present)
-    if args.participant_label != None:
+    if args.participant_label is not None:
         for p in args.participant_label:
-            to_deface.extend(glob(os.path.join(args.bids_dir, f'sub-{p}', "ses-*")))
+            to_deface.extend(list(args.bids_dir.joinpath(f'sub-{p}').glob("ses-*")))
```

Comment:
glob 1. These comments are just making sure this was intentional.

Reply:
Yes, this was intentional just to stay consistent with usage of pathlib library coz all the other scripts use pathlib.Path objects. And I think it's a bit more readable too.